keps/sig-node/2371-cri-pod-container-stats/kep.yaml

```diff
@@ -0,0 +1,23 @@
+kep-number: 2371
+title: cAdvisor-less, CRI-full Container and Pod Stats
+authors:
+  - "@haircommander"
+  - "@bobbypage"
+owning-sig: sig-node
+participating-sigs:
+reviewers:
+  - "@derekwaynecarr"
+  - "@mrunalp"
```

Comment:
You can add me as a reviewer for sig-instrumentation

Reply:
added!